docs/resources/policy_transit_gateway.md

```diff
@@ -8,7 +8,7 @@ description: A resource to configure a Transit Gateway.
 
 This resource provides a method for the management of a Transit Gateway.
 
-This resource is applicable to NSX Policy Manager.
+This resource is applicable to NSX Policy Manager and is supported with NSX 9.0.0 onwards.
```

Comment:
I think a clarification might be worth here.

In 9.0, NSX will not allow users to create TGWs.
However, TF users could import a project's default TGW and use the resource to update it.

Starting with NSX 9.1 users can also create/delete TGWs.

Also the high_availability_config and span attributes have been added in 9.1